include/tvm/runtime/packed_func.h

```diff
@@ -828,6 +840,11 @@ class TVMRetValue : public TVMPODValue_ {
             kNodeHandle, *other.template ptr<NodePtr<Node> >());
         break;
       }
+      case kObject: {
```

Comment:
@tqchen having trouble getting this code working

Reply:
please confirm again if this works or now